src/main/resources/db/migration/V1_13__Create_case_data_tables.sql

```diff
@@ -1,4 +1,4 @@
-create table if not exists darts.moj_reporting_restrictions
```

Comment:
Its prepends "darts" in Barrys scripts.  Is there a reason we should't?.

Reply:
as discussed, its to enable flexibility, and currently build breaks if they are in.